test-app/tests/integration/components/headless-form-reset-test.gts

```diff
@@ -153,4 +154,159 @@ module('Integration Component HeadlessForm > Reset', function (hooks) {
         .doesNotExist('form.submissionState is resetted');
     });
   });
+
+  module('reset action', function () {
```

Comment:
This is duplicating the same tests as for the reset button. Not sure if/how we should try to DRY this, but I thought for tests some duplication is ok, instead of trying to be too clever in tests?

Reply:
> for tests some duplication is ok, instead of trying to be too clever in tests

100% agree!